index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>@Gopxfs Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+<body>
+  <div class ="page-flex">
```

Comment:
- [ ] Your portfolio is looking great, good job! It would be even better if you removed the surrounding white margin which is not part of the Figma template:

![Captura de pantalla 2022-04-20 a la(s) 11 57 35 p  m](https://user-images.githubusercontent.com/83717285/164375531-aa305c9f-43d4-4dc4-be14-47e2b77a849d.png)